src/tests/benchmark/darktable-bench

```diff
@@ -66,6 +67,13 @@ def locate_program(program):
             return build+program
          if VERBOSE:
             print(f'  did not find {program} in {build}')
+      if platform.system() == 'Darwin':
+         macpath = '/Applications/darktable.app/Contents/MacOS/'
+         program = 'darktable-cli'
```

Comment:
The user could have specified a different executable name than 'darktable-cli'.  This forces the name regardless of commandline args.

Reply:
Yes you're right. Thanks. I will fix it.